### DISCUSSION TITLE
Problems connecting github to vs studio 15

### DISCUSSION THREAD

**Discussion by ghost** [community/community#37140] [Other Feature Feedback, Questions, & Ideas]
I really need to use studio 15 right now, I have a couple of live projects that I want to put on the gith. What should I do, why can't I log into my profile? Here in 22 studios there are no such problems.

![1](https://user-images.githubusercontent.com/64988751/197593620-8ed4a167-8c06-4be7-bfa6-4b91667ca579.png)
![2](https://user-images.githubusercontent.com/64988751/197593623-c70be165-41ab-43c8-8e9c-498043fc0feb.png)

How can I deal with this problem? I can’t switch completely to studio 22 yet, how can I upload my projects from studio 15 to github in another way?

**Comment by ossanna16** [comment]
Hi there @Antsently and welcome to our community! Thank you for asking a great question 🙂

**Reply by ghost** [reply]
> Всем привет@Antsentlyи добро пожаловать в наше сообщество! Спасибо, что задали отличный вопрос🙂

Great, I'm waiting for an explanation of my problem.
